indigo_app/templates/indigo_api/work_form.html

```diff
@@ -274,7 +275,18 @@ <h5 class="card-header">Commencement and assent</h5>
                   <input type="checkbox" class="form-check-input" id="commencement_date_unknown">
                   <label for="commencement_date_unknown">Commencement date is unknown</label>
                 </div>
-                <p class="form-text text-muted">Check this box when a (usually old) work has come into force, but the exact commencement date is not known.</p>
+                <p class="form-text text-muted">Check this box when a work has come into force, but the exact commencement date is unknown or open to interpretation.</p>
+
+                <div id="commencement_note">
+                  <div class="form-row">
```

Comment:
The `from-row` is only necessary if it has a `col-*` as an immediate child. (I know further up it appears erroneously)